src/main/java/org/broadinstitute/consent/http/models/DataAccessRequestData.java

```diff
@@ -26,7 +26,7 @@ public class DataAccessRequestData {
             "profileName", "pubmedId", "scientificUrl", "eraExpiration", "academicEmail",
             "eraAuthorized", "nihUsername", "linkedIn", "orcid", "researcherGate", "datasetDetail",
             "datasets", "datasetId", "validRestriction", "restriction", "translatedUseRestriction",
-            "createDate", "sortDate");
+            "createDate", "sortDate", "additionalEmail", "checkNotifications" );
```

Comment:
This is a non-issue, but I thought I would describe it for posterity. `DEPRECATED_PROPS` is used to create a simplified version of the DAR where it filters out all of these values. I initially thought we would still need to do that filtering, but it turns out that since you removed the getters/setters, these fields are never populated in any db call. So, nice work.